backend/test/integration/run_tests_locally.sh

```diff
@@ -32,7 +32,22 @@ case "$response" in
         ;;
 esac
 
-echo "Starting integration tests..."
-command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true"
-echo $command "$@"
-$command "$@"
+if [ "$1" == "postgres" ]; then
+    echo "Starting PostgreSQL DB port forwarding..."
+    kubectl -n "$NAMESPACE" port-forward svc/postgres-service 5432:5432 --address="127.0.0.3" & PORT_FORWARD_PID=$!
+    # wait for kubectl port forward
+    sleep 10
+    echo "Starting integration tests..."
+    command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true -runPostgreSQLTests=true"
+    echo $command "$@"
+    $command "$@"
+else 
+    echo "Starting MySQL DB port forwarding..."
+    kubectl -n "$NAMESPACE" port-forward svc/mysql 3306:3306 --address=localhost & PORT_FORWARD_PID=$!
+    # wait for kubectl port forward
+    sleep 10
+    echo "Starting integration tests..."
+    command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true"
+    echo $command "$@"
+    $command "$@"
```

Comment:
NIT: How about using `set -x` at the top of file to print command automatically, instead of echoing here?

Reply:
Thank you! I updated the file.